Realm/Realm/DataBinding/RealmObjectTypeDelegator.cs

```diff
@@ -31,19 +31,16 @@ internal RealmObjectTypeDelegator(Type type) : base(type)
         {
         }
 
-        public override PropertyInfo GetDeclaredProperty(string name)
+        protected override PropertyInfo GetPropertyImpl(string name, BindingFlags bindingAttr, Binder binder, Type returnType, Type[] types, ParameterModifier[] modifiers)
         {
-            return _propertyCache.GetOrAdd(name, n =>
+            var result = base.GetPropertyImpl(name, bindingAttr, binder, returnType, types, modifiers);
+            var wovenAttribute = result.GetCustomAttribute<WovenPropertyAttribute>();
```

Comment:
```suggestion
            var wovenAttribute = result?.GetCustomAttribute<WovenPropertyAttribute>();
```
This way if the binding expression has a typo (e.g. `{Binding FulName}`) we won't crash because we tried to look up a non-existing property. I think there was an issue about that somewhere

Reply:
Makes sense! Corrected